dagfactory/dagbuilder.py

```diff
@@ -267,10 +266,21 @@ def make_timetable(timetable: str, timetable_params: Dict[str, Any]) -> Timetabl
             raise DagFactoryException(f"Failed to import timetable {timetable} due to: {err}") from err
         try:
             schedule: Timetable = timetable_obj(**timetable_params)
-        except Exception as err:
+        except Exception as err:  # pragma: no cover
```

Comment:
Would it be possible for us to capture a more specific exception here, instead of `Exception`? I understand it was like this before, but it may be an opportunity to improve this.

Reply:
I noticed this pattern appears in a few places throughout the repo. Would it be okay if I addressed it in a separate PR? I've created an issue to track it here: https://github.com/astronomer/dag-factory/issues/410.